homework5/secondTask.js

```diff
@@ -0,0 +1,44 @@
+const buttonToggle = document.querySelector(".button--toggle");
+const container = document.querySelector(".section");
+const infoMessage = document.querySelector(".info-message")
+
+let pageState = {};
+document.addEventListener('DOMContentLoaded', () => {
+    pageState = localStorage.getItem('pageState') !== null
+        ? JSON.parse(localStorage.getItem('pageState'))
+        : {
+            isTurnedOn: false,
+            lastTimeTurnedOn: null,
+            lastTimeTurnedOff: null
+        };
+
+    updatePage();
+});
+
+buttonToggle.addEventListener("click", function (e){
```

Comment:
```suggestion
buttonToggle.addEventListener("click", function (event){
```
краще уникати односимвольних назв всюди де це можливо